README.md

```diff
@@ -175,7 +176,20 @@ You can create up to 5,000 custom labels for your organization, and they can be
 
 #### Update Labels
 
-To update the Custom Labels metadata, go to the `force-app/main/default/labels directory`. The labels are available in the `CustomLabels.labels-meta.xml` file. For an example of a CustomLabels definition, see the [Metadata API Dev Guide](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_customlabels.htm).
+To update the Custom Labels metadata, go to the `force-app/main/default/labels` directory. The labels are available in the `lightning.labels-meta.xml` file. For an example of a CustomLabels definition, see the [Metadata API Dev Guide](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_customlabels.htm).
+
+Add labels for your custom components in the `lightning.labels-meta.xml` file.
+
+```xml
+<labels>
+    <categories>lightning</categories>
+    <fullName>MyCard_cardTitle</fullName>
+    <language>en_US</language>
```

Comment:
Do we have guidelines for language values? Maybe something like "Use an ISO language code to describe your label's language. For more information, see https://www.iso.org/iso-639-language-codes.html"?
@sgesin , not sure what we should point people to.

Reply:
The Metadata API Dev Guide that we link to just above this mentions this, and links to https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_translations.htm#translation_languages